hash/m20.xml

```diff
@@ -159,137 +199,150 @@ license:CC0-1.0
 		<description>Olivetti Z8000 Assembler 2.1</description>
 		<year>19??</year>
 		<publisher>Olivetti</publisher>
 
 		<part name="flop1" interface="floppy_5_25">
 			<dataarea name="flop" size="286720">
-				<rom name="asm_r2.1.a.img" size="286720" crc="90b0fb9e" sha1="b250ff32d40293da446ce96b513b0000fb7814d1"/>
+				<rom name="asm_r2.1.a.img" size="286720" crc="90b0fb9e" sha1="b250ff32d40293da446ce96b513b0000fb7814d1" status="baddump"/>
 			</dataarea>
 		</part>
 	</software>
 
-	<software name="dietasm">
+	<software name="dietasm" supported="no"><!-- Missing FM track0, boot with -ramsize 512k -->
 		<description>ASM Utilities?</description>
 		<year>19??</year>
 		<publisher>&lt;unknown&gt;</publisher>
 
 		<part name="flop1" interface="floppy_5_25">
 			<dataarea name="flop" size="286720">
-				<rom name="dieters_asm.img" size="286720" crc="27533432" sha1="9500d75444b84b20c41300f30b9c840e575d6542"/>
+				<rom name="dieters_asm.img" size="286720" crc="27533432" sha1="9500d75444b84b20c41300f30b9c840e575d6542" status="baddump"/>
 			</dataarea>
 		</part>
 	</software>
 
-	<software name="pascal" supported="no">
+	<software name="pascal" supported="no"><!-- Missing FM track0 (all), boot with -ramsize 512k -->
+		<!-- launch with > pas? -->
 		<description>Olivetti Pascal</description>
 		<year>19??</year>
 		<publisher>Olivetti</publisher>
 
 		<part name="flop1" interface="floppy_5_25">
 			<dataarea name="flop" size="286720">
-				<rom name="pascal1_4.img" size="286720" crc="fe422ff7" sha1="e9d341e3d4f57ba8bcf249fc3d261a1867c92892"/>
+				<rom name="pascal1_4.img" size="286720" crc="fe422ff7" sha1="e9d341e3d4f57ba8bcf249fc3d261a1867c92892" status="baddump"/>
 			</dataarea>
 		</part>
 		<part name="flop2" interface="floppy_5_25">
 			<dataarea name="flop" size="286720">
-				<rom name="pascal2_4.img" size="286720" crc="2131b9b8" sha1="a51bc215b4d22a0b162db6a4f970a32430a78688"/>
+				<rom name="pascal2_4.img" size="286720" crc="2131b9b8" sha1="a51bc215b4d22a0b162db6a4f970a32430a78688" status="baddump"/>
 			</dataarea>
 		</part>
 		<part name="flop3" interface="floppy_5_25">
 			<dataarea name="flop" size="286720">
-				<rom name="pascal3_4.img" size="286720" crc="7a2ffa83" sha1="1b9e760a2bd38a87273387d37b7e8c1e6400465d"/>
+				<rom name="pascal3_4.img" size="286720" crc="7a2ffa83" sha1="1b9e760a2bd38a87273387d37b7e8c1e6400465d" status="baddump"/>
 			</dataarea>
 		</part>
 		<part name="flop4" interface="floppy_5_25">
 			<dataarea name="flop" size="286720">
-				<rom name="pascal4_4.img" size="286720" crc="accf7de5" sha1="4c02cea1a9072ef610085688e57a80b48e82d502"/>
+				<rom name="pascal4_4.img" size="286720" crc="accf7de5" sha1="4c02cea1a9072ef610085688e57a80b48e82d502" status="baddump"/>
 			</dataarea>
 		</part>
 	</software>
 
 	<software name="dbms" supported="partial">
+		<!-- launch with > ba "dbms.bas" -->
 		<description>Olivetti Data Base Management System</description>
```

Comment:
It’s better to put instructions for launching in `usage` `info` elements so it’s visible in the UI.